### PR TITLE
add interior-only smoke tests

### PR DESCRIPTION
**Pull request by granzoto**
@fgiorgetti @nicob87  could you please review ?

Signed-off-by: Renato Granzoto <rgranzot@redhat.com>

### CODE REVIEW

File: test/smoke/interior/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                  v                             v
+           -----------------            -----------------
+           + interior-west +   ----->   + interior-east +
+           -----------------            -----------------
+                  ^                              ^
+---------------   |                             |   ---------------
++ edge-west-2 + ---                             --- + edge-east-2 +
+---------------                                     ---------------
+```
+
```
Comment (fgiorgetti) [inline test/smoke/interior/README.md:21]: guess we need to update this diagram...

File: test/smoke/interior/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Interior and Edge meshes
+
+Deploys two interior router meshes (running in same namespace, within the same
+context). Two edge routers will connect to each interior mesh.
```
Comment (fgiorgetti) [inline test/smoke/interior/README.md:4]: modify this info here to be accurate with this new scenario

**Review by fgiorgetti — CHANGES_REQUESTED** [review]

File: test/smoke/interior/README.md (hunk 1 above)
Comment (granzoto) [reply test/smoke/interior/README.md:13]: ok, Topology updated

File: test/smoke/interior/README.md (hunk 2 above)
Comment (granzoto) [reply test/smoke/interior/README.md]: ok, Description updated

File: test/smoke/interior/interior_suite_test.go [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	"github.com/rh-messaging/shipshape/pkg/framework/log"
+	"os"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
+	testcommon.Initialize(m)
+}
+
+func TestInterioredge(t *testing.T) {
```
Comment (fgiorgetti) [inline test/smoke/interior/interior_suite_test.go:14]: One more Renato. Lets change it as it might be helpful when dealing with reporting.

**Review by fgiorgetti — CHANGES_REQUESTED** [review]

File: test/smoke/interior/interior_suite_test.go (hunk 3 above)
Comment (granzoto) [reply test/smoke/interior/interior_suite_test.go]: Changed

**Review by nicob87 — APPROVED** [review]
Nice!

**Review by fgiorgetti — APPROVED** [review]

**Pull request merged**